### ISSUE TITLE
Company Email address

### ISSUE THREAD

**Issue by mcchrish** [gwillws/gwillws.github.io#16]
It will be nice if can use email address like @gwill.org
It looks more professional than @gmail.com (which is your personal email address)

**Comment by gwillws** [comment]
yes. please access our domain. the account details is in evernote

**Comment by mcchrish** [comment]
I already made email addresses for each of us. You can check your email details in evernote. The passwords are autogenerated but you can change it by login into dreamhost as admin and change it from there.
I was able to link it to my gmail too. So I can receive and reply messages using gmail.
Here are the steps: https://jryven.com/gmail-email-alias-dreamhost/
(Note that I think dreamhost will not automatically forward emails received by gwill.org to you gmail, so what I did was I put my gmail address to the forward list and that fix it)

**Comment by gwillws** [comment]
thanks

**Comment by gwillws** [comment]
i think it should be info@gwillws.org
